src/controllers/users.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const { userService } = require('../services/users.js');
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = userService.create(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const remove = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  userService.remove(+userId);
+
+  res.sendStatus(204);
+};
+
+const update = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  const { name } = req.body;
+
+  userService.update(userId, name);
+
+  res.send(foundUser);
```

Comment:
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/84296041/c1905221-5322-45b5-810e-6175ab8c296b)

Send the error with status code 400, when you don't get a userId.
Send the error with status code 404, when you don't find a user.
Send the response with status code 200, when everything is fine.